app/views/application/_sidebar.html.erb

```diff
@@ -16,7 +16,7 @@
 
   <%= yield :sidebar %>
 
-  <% if current_user %>
+  <% unless current_user.try(:teams).blank? %>
```

Comment:
You also can use `if current_user.try(:teams).present?` to keep this condition easier to read